js_modules/dagster-ui/packages/ui-core/src/assets/BackfillPreviewModal.tsx

```diff
@@ -112,7 +112,11 @@ export const BackfillPreviewModal = ({
                     {partitions ? (
                       <TargetPartitionsDisplay targetPartitions={partitions} />
                     ) : (
-                      <Spinner purpose="body-text" />
+                      loading ? (
+                        <Spinner purpose="body-text" />
+                      ) : (
+                        "No partitions available to materialize"
```

Comment:
Alt options:
- No partitions in selected range
- No partitions in selected range available to materialize 
- None